wsl_test.go

```diff
@@ -1718,6 +1718,55 @@ func TestWithConfig(t *testing.T) {
 			}`),
 			expectedErrorStrings: []string{reasonOnlyCuddleIfWithAssign},
 		},
+		{
+			description: "allow cuddling of declaration (#83)",
+			code: []byte(`package main
+
+			func main() {
+				var first = 1
+				var second = 2
```

Comment:
I'd add this so that it's obvious this is possible now:
```go
third := 3
```
I think it should also be put to the test `allow cuddle var` at line ~1311

Reply:
Yeah that's a good idea. It's cuddled with a AssignStmt at line 1737 but it's not assigning a new variable so probably not as obvious.